src/python/pants/backend/python/interpreter_cache.py

```diff
@@ -102,14 +102,15 @@ def select_interpreter_for_targets(self, targets):
             raise self.UnsatisfiableInterpreterConstraintsError(
                 dedent(
                     """\
-        Unable to detect a suitable interpreter for compatibilities: {} (Conflicting targets: {})
-
-        Pants detected these interpreter versions on your system: {}
-
-        Possible ways to fix this:
-        * Modify your Python interpreter constraints by following https://www.pantsbuild.org/python_readme.html#configure-the-python-version.
-        * Ensure the targeted Python version is installed and discoverable.
-        * Modify Pants' interpreter search paths via --pants-setup-interpreter-search-paths.""".format(
+                    Unable to detect a suitable interpreter for compatibilities: {} (Conflicting targets: {})
+            
+                    Pants detected these interpreter versions on your system: {}
+            
+                    Possible ways to fix this:
+                    * Modify your Python interpreter constraints by following https://www.pantsbuild.org/python_readme.html#configure-the-python-version.
+                    * Ensure the targeted Python version is installed and discoverable.
+                    * Modify Pants' interpreter search paths via --pants-setup-interpreter-search-paths.
```

Comment:
Will this not add an extra newline and indent at the end of the string literal?

Reply:
You're right that this will add a new line. Good catch! What do you mean about an indent? Because we use `dedent()`, I think there will be only a new line.

 I'm also not sure if the newline is a bad thing per se.

Reply:
Didn't read the `dedent`, sorry. I'm not sure that the extra space changes anything, but it definitely changes the formatting of the string. Fine with fixing it in a follow-up though.